src/toil/batchSystems/slurm.py

```diff
@@ -127,26 +130,32 @@ def _getJobDetailsFromScontrol(self, slurmJobID):
                     'show',
                     'job',
                     str(slurmJobID)]
-    
+
             process = subprocess.Popen(args, stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
-    
+            rc = process.wait()
+
+            if rc != 0:
+                # no accounting system or some other error
+                return (None, -999)
+
             job = dict()
             for line in process.stdout:
+                logger.debug("%s output %s", args[0], line)
                 values = line.decode('utf-8').strip().split()
-    
+
                 # If job information is not available an error is issued:
                 # slurm_load_jobs error: Invalid job id specified
                 # There is no job information, so exit.
                 if len(values)>0 and values[0] == 'slurm_load_jobs':
                     return (None, None)
```

Comment:
I think the early exit `return (None, -999)` might be related to [an issue where the toil never finishes](https://github.com/DataBiosphere/toil/pull/2959#issuecomment-584207037) on this branch.
The comment at the bottom seems to say that it should `return (None, None)`.

For completed jobs slurm responds with the following for scontrol:
```
$ scontrol show job 15671080
slurm_load_jobs error: Invalid job id specified
```

Reply:
thanks, yes only _getJobDetailsFromSacct() should return -999 magic number..